README.md

```diff
@@ -16,6 +16,9 @@ List of links to the various checkers out there on the web for sites, domains, s
 | URL | Description | 💳 |
 | --- | ----------- | -- |
 | https://www.browserstack.com/ | Test your website for cross browser compatibility on real browsers. Instant access to multiple desktop and mobile browsers. | Paid |
+| http://browsershots.org | Test websites on many browser, platforms and devices. | Free |
```

Comment:
As per the [Contribution Guide](https://github.com/Brunty/awesome-checker-services/blob/master/CONTRIBUTING.md) can you add them alphabetically?

> Within a section, please ensure new services are added alphabetically (by name / domain) in the list.